networkx/readwrite/gml.py

```diff
@@ -216,12 +216,12 @@ def parse_gml(lines, label='label', destringizer=None):
 
     label : string, optional
         If not None, the parsed nodes will be renamed according to node
-        attributes indicated by ``label``. Default value: ``'label'``.
+        attributes indicated by `label`. Default value: `'label'`.
 
     destringizer : callable, optional
         A destringizer that recovers values stored as strings in GML. If it
-        cannot convert a string to a value, a ``ValueError`` is raised. Default
-        value : ``None``.
+        cannot convert a string to a value, a `ValueError` is raised. Default
```

Comment:
Exceptions can (and should) be rendered as class names and linked as `:exc:ValueError`.